apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-without-translations-presenter.tsx

```diff
@@ -0,0 +1,22 @@
+import { IMultilingualTextItem } from '@coscrad/api-interfaces';
+import { Paper, styled } from '@mui/material';
+import { MultilingualTextItemPresenter } from './multilingual-text-item-presenter';
+
+const MultilingualTextHeaderBox = styled(Paper)({
+    paddingLeft: '8px',
+});
+
+interface MultilingualTextWithoutTranslationsProps {
+    primaryMultilingualTextItem: IMultilingualTextItem;
+}
+
+export const MultilingualTextWithoutTranslations = ({
```

Comment:
I'm not sure about this approach, but I'll defer to you. See comment above.

Reply:
See above comment I had to add to Aaron's comment (only gave me the Edit option).